table-fixed-header.js

```diff
@@ -1,67 +1,79 @@
-(function($) {
+(function ($) {
 
-$.fn.fixedHeader = function (options) {
- var config = {
-   topOffset: 38
-   //bgColor: 'white'
- };
- if (options){ $.extend(config, options); }
+  $.fn.fixedHeader = function (options) {
```

Comment:
unneccessary diff on untouched code. Even if you have a preference on number of spaces, a good rule is to never do such formatting along with code changes, it masks the code changes. If done in separate commits it would be acceptable, even though the whole PR is more difficult to review. Please create PR of the code changes only, thanks for contributing.

Reply:
Sorry,I didn't notice that.